dll/win32/msgina/shutdown.c

```diff
@@ -463,12 +675,24 @@ ShutdownOnInit(
     INT idx, count, i;
     WCHAR szBuffer[256];
     WCHAR szBuffer2[256];
+    HDC hdc;
+    long lfHeight;
+
+    /* Create font for the IDC_TURN_OFF_STATIC static control */
+    hdc = GetDC(hDlg);
+    lfHeight = -MulDiv(12, GetDeviceCaps(hdc, LOGPIXELSY), 68);
+    ReleaseDC(hDlg, hdc);
+    pContext->hfFont = CreateFontW(lfHeight, 0, 0, 0, 550, FALSE, 0, 0, 0, 0, 0, 0, 0, L"MS Shell Dlg");
```

Comment:
I'm not fond of hardcoding values in the `cWeight` argument when there are already defined specific values by the API.
https://docs.microsoft.com/en-us/windows/win32/api/wingdi/nf-wingdi-createfontw